packages/babel-check-binary-expressions/src/index.js

```diff
@@ -0,0 +1,26 @@
+import isNumber from "lodash/isNumber";
+import isString from "lodash/isString";
```

Comment:
I think we can void the `lodash` dependency here, you can replace it with:

`typeof x != "number"` and `typeof x != "string"`.